src/settings.ui

```diff
@@ -26,7 +26,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>3</number>
+      <number>1</number>
```

Comment:
@csharpee why is this change needed?

Reply:
I didn't change it, for some reason clicking tabs changes the number in Qt. I can manually edit this with Geany :genie: ..Qt doesn't let us manually edit the code